ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -152,3 +152,11 @@ tests:
     TEST_SUITE=openshift/conformance/parallel run-tests
   openshift_installer:
     cluster_profile: aws
+
```

Comment:
nit: I don't see a need to split this out from the rest of the the tests defined above.  Can we drop this empty line and alphabetize the new entry into the block above?

Reply:
Fixed in 603449306